packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -250,24 +252,21 @@ function* confirmEditTrack(
           throw new Error('No userId set, cannot edit track')
         }
 
-        const { blockHash, blockNumber } = yield* call(
-          audiusBackendInstance.updateTrack,
-          userId,
-          trackId,
-          { ...formFields },
+        if (!userId) return
+        const coverArtFile =
+          formFields.artwork && 'file' in formFields.artwork
+            ? formFields.artwork.file
+            : undefined
+
+        yield* call([sdk.tracks, sdk.tracks.updateTrack], {
+          userId: Id.parse(userId),
+          trackId: Id.parse(trackId),
+          coverArtFile: coverArtFile
+            ? artworkFileToSDK(coverArtFile)
+            : undefined,
+          metadata: trackMetadataForUploadToSdk(formFields),
           transcodePreview
-        )
-
-        const confirmed = yield* call(
```

Comment:
Oh interesting. I wonder if we even need to do all of this inside confirmer at this point? If SDK is doing write confirmation internally as part of the `manageEntity` call. I think we could simplify this saga.

Reply:
There is still value to using our confirmer sagas, we've just shifted the actual confirmation call into the sdk. The confirmer sagas orchestrate multiple confirmation requests, handle the state that warns the user about refreshing, and allows us to debug with the confirmer preview